.ci/validation.sh

```diff
@@ -42,7 +42,7 @@ check-missing-pitests)
 
   list=($(cat pom.xml | \
     xmlstarlet sel --ps -N pom="http://maven.apache.org/POM/4.0.0" \
-    -t -v '//pom:profile[./pom:id[contains(text(),'pitest')]]//pom:targetClasses/pom:param'))
+    -t -v '//pom:profile[./pom:id[contains(text(),'"'pitest'"')]]//pom:targetClasses/pom:param'))
```

Comment:
I do not understand why we need all the extra quotes here; why can’t we do https://github.com/checkstyle/checkstyle/pull/12724#discussion_r1102266548 ?

Reply:
Shellcheck is not a substitute for humans :) it only sees `'text_in_quotes'text_not_in_quotes'text_in_quotes'` and realizes that the `text_not_in_quotes` is not in the quotes, which is correct. In bash, using `”` allows us to use `'` inside, where another `'` just closes the previous one.

Reply:
Note that the shellcheck example is a very specific case that doesn’t apply here: in their example, they do not want the `$host` interpolated. This is why they avoided doing what I am suggesting here. We have no variables for interpolation/expansion here, so we should be fine to enclose this expression in double quotes.

Reply:
Done.